src/main/java/cms/gov/madie/measure/utils/SequenceGeneratorUtil.java

```diff
@@ -0,0 +1,17 @@
+package cms.gov.madie.measure.utils;
+
+import cms.gov.madie.measure.repositories.GeneratorRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class SequenceGeneratorUtil {
```

Comment:
I think this has to be a service and not util.

Reply:
thanks, I have removed the util itself